ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -102,16 +115,20 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
     ///   - containerWidth: The width of the view's super view (e.g. the table view's width).
     /// - Returns: a value representing the calculated height of the view.
     @objc public class func height(style: Style, title: String, titleNumberOfLines: Int = 1, containerWidth: CGFloat = .greatestFiniteMagnitude) -> CGFloat {
-        let titleWidth = containerWidth - (Constants.horizontalMargin + TableViewHeaderFooterView.titleRightOffset())
-        let titleHeight = title.preferredSize(for: Constants.titleTextStyle.font, width: titleWidth, numberOfLines: titleNumberOfLines).height
-
         let verticalMargin: CGFloat
+        var font: UIFont = Constants.titleTextStyle.font
         switch style {
         case .header, .footer:
             verticalMargin = Constants.titleDefaultTopMargin + Constants.titleDefaultBottomMargin
+        case .headerPrimary:
```

Comment:
we should document what headerPrimary means now that there are two different header style.